lib/nodes/file.cpp

```diff
@@ -127,6 +132,15 @@ int villas::node::file_parse(NodeCompat *n, json_t *json) {
       throw RuntimeError("Invalid value '{}' for setting 'epoch'", epoch);
   }
 
+  if (read_mode) {
+    if (!strcmp(read_mode, "all")) {
+      n->logger->debug("read mode: all set");
+      f->read_mode = file::ReadMode::READ_ALL;
```

Comment:
```suggestion
```

Reply:
Let put this into `file_print()` (`Node::getDetails()`).

Here: https://github.com/VILLASframework/node/blob/aaece4c1b8e27ca883b1e5d0b55e6721ada6d2b8/lib/nodes/file.cpp#L147